src/__tests__/testScreen.py

```diff
@@ -196,4 +207,16 @@ def maybeMakeExpectedDir():
             os.makedirs(EXPECTED_DIR)
 
 if __name__ == '__main__':
+    # monkey patch clearLine to clear the input region, not print spaces
+    # this matches the intent of the function and makes sure we don't
+    # generate empty spaces in the output for comparing against the expected
+    # input
+
+    def clearLine(self, y):
+        (minx, miny, maxx, maxy) = self.getChromeBoundaries()
+        for x in range(minx, maxx):
+            self.stdscr.output[(x, y)] = ('', 1)
+
+    screenTestRunner.choose.screenControl.Controller.clearLine = clearLine
```

Comment:
:see_no_evil: this seems a bit hacky too. I know maybe its valid python but usually some kind of more obvious and more explicit dependency injection is better than overwriting the functionality of the module to be tested